site/features.md

```diff
@@ -6,44 +6,70 @@ menu_order: 15
 
-Nodes can be filtered by various properties, for example in the Container view, you can filter by System Containers vs Application Containers or by Stopped vs. Running Containers. Scope can also display various metrics such as CPU and Memory usage in the nodes, for example allowing you to easily find the container using the most CPU or memory.  Mouse-over any of the nodes to view its filtered metric at a glance.
+!['CPU usage indicator in Weave Scope'](images/hosts-cpu.png)
 
-###<a name="powerful-search"></a> Powerful Search
+##<a name="powerful-search"></a> Powerful Search
 
-Powerful search capability enables you to find node types, containers and processes by name, label or even path.  The search functionality supports simple operands so that you can for example, find processes consuming a certain Memory limit or nodes using too much CPU. You can also stack filters to create custom, complex search criterion. 
+Powerful search capability enables you to find node types, containers and processes by name, label or even path.  The search functionality supports simple operands so that you can for example, find processes consuming a certain Memory limit or nodes using too much CPU. Search terms may also be stacked to create custom, complex search criterion. See the online help for a full list of allowable operands.
 
-!['Stack Filters'](images/search-python.png)
+!['Powerful Search'](images/search.png)
 
 ##<a name="real-time-app-and-container-metrics"></a>Real-time App and Container Metrics
 
-View contextual metrics, tags and metadata for your containers.  Effortlessly navigate between processes inside your container to the hosts that your containers run on, arranged in expandable, sortable tables.
+View contextual metrics, tags and metadata for your containers by clicking on a node to display its details panel. Drilldown on processes inside your container to the hosts that your containers run on, arranged in expandable, sortable tables.
 
 Choose an overview of your container infrastructure, or focus on a specific microservice. Identify and correct issues to ensure the stability and performance of your containerized applications.
 
-##<a name="interact-with-and-manage-containers"></a>Interact With and Manage Containers
+##<a name="interact-with-and-manage-containers"></a>Troubleshoot and Manage Containers
 
-Interact with your containers directly: pause, restart and stop containers without having to leave the Scope browser window.
+Click on a container, pod or host to view the controls that allow you to: pause, restart, stop and delete without having to leave the Scope browser window. Logs of selected containers or pods (if you are running in Kubernetes) can also be viewed by clicking the terminal icon.
```

Comment:
I would place _(if you are running in Kubernetes)_ after the first mention of pod